src/Microsoft.Health.SqlServer/Configs/SqlClientRetryOptions.cs

```diff
@@ -24,5 +25,34 @@ public class SqlClientRetryOptions
         NumberOfTries = 5,
         DeltaTime = TimeSpan.FromSeconds(1),
         MaxTimeInterval = TimeSpan.FromSeconds(20),
+
+        // Default errors copied from src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Reliability/SqlConfigurableRetryFactory.cs .
+        TransientErrors = new HashSet<int>
```

Comment:
Aren't these being included by default if errors are left unspecified? So we don't need to explicit specify these values.

Reply:
Yes, but there's no way to add more, you can only replace with new set of errors. Everything is internal, sealed, protected, private.

Reply:
Oh, I see. Sorry for the misunderstanding. I totally missed that